solr/core/src/java/org/apache/solr/search/facet/FacetProcessor.java

```diff
@@ -320,35 +320,13 @@ void resetStats() throws IOException {
   }
 
   int collect(DocSet docs, int slot) throws IOException {
-    int count = 0;
```

Comment:
This diff was in the original patch I started with, it's removing a load of code that's identical to another method, and calling that instead.